paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/cdc/mysql/MySqlSyncDatabaseAction.java

```diff
@@ -105,6 +110,10 @@ public class MySqlSyncDatabaseAction extends ActionBase {
     @Nullable private final Pattern includingPattern;
     @Nullable private final Pattern excludingPattern;
     private final Map<String, String> tableConfig;
+    private final String includingTables;
+    private final List<String> excludedTables;
```

Comment:
excludedTables can be local field in `build`, we can introduce `getMySqlSchemaList(List<String> excludedTables)`.